doc/notebooks/jupyter_integration.ipynb

```diff
@@ -78,22 +78,21 @@
     "# Let's display the DTM of our sample area to ensure all's working\n",
     "\n",
     "# Set computational region to the study area.\n",
-    "gs.parse_command('g.region', raster='elevation', flags='pg')\n",
+    "gs.parse_command(\"g.region\", raster=\"elevation\", flags=\"pg\")\n",
     "\n",
     "# Draw elevation (DTM) to get an overview of the area.\n",
-    "gs.run_command('r.colors', map='elevation', color='elevation')\n",
-    "gs.run_command('d.erase')\n",
-    "gs.run_command('d.rast', map='elevation')\n",
-    "gs.run_command('d.legend', raster='elevation', at=(65, 90, 85, 90), fontsize=15, flags='b', title='DTM')\n",
-    "Image(filename='map.png')"
+    "gs.run_command(\"r.colors\", map=\"elevation\", color=\"elevation\")\n",
+    "gs.run_command(\"d.erase\")\n",
+    "gs.run_command(\"d.rast\", map=\"elevation\")\n",
+    "gs.run_command(\"d.legend\", raster=\"elevation\", at=(65, 90, 85, 90), fontsize=15, flags=\"b\", title=\"DTM\")\n",
+    "Image(filename=\"map.png\")"
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
-   "language": "python",
-   "name": "python3"
+   "name": "python3",
+   "display_name": "Python 3.8.3 64-bit ('base': conda)"
```

Comment:
I'm not sure how to handle those in the future, but right now, the best practice seem to keep here something simple without the conda complexities (if for no other reason, it is meant to be used also without conda), so revert this piece.